src/component/plotly-dash-preview/parse.js

```diff
@@ -24,6 +24,13 @@ function parse (body, opts, sendToRenderer) {
   }
 
   result.pdfOptions = body.pdf_options || {}
+  if (!body.loading_selector && !body.timeout) {
```

Comment:
We should use utils `isPositiveNumber` for `timeout` and `isNonEmptyString` for `loading_selector`.

Oh and I would vote for just `selector` instead of `loading_selector` which is unnecessarily verbose here I think.